sdk/developersigning/CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributing
+
```

Comment:
This doesn't seem to add any additional information to the repository contributing docs.   We should remove it.